dandi/metadata.py

```diff
@@ -347,7 +347,13 @@ def extract_species(metadata):
             value = "Human"
         elif "norvegicus" in value:
             value_id = "http://purl.obolibrary.org/obo/NCBITaxon_10116"
-            value = "Brown rat"
+            value = "Rat; Norway rat; rats; Brown rat"
+        elif "rattus rattus" in value:
+            value_id = "http://purl.obolibrary.org/obo/NCBITaxon_10117"
+            value = "Black rat; Roof rat; House rat"
+        elif "rat" in value:
+            value_id = "http://purl.obolibrary.org/obo/NCBITaxon_10116"
+            value = "Rat; Norway rat; rats; Brown rat"
```

Comment:
I guess check for `norvegicus` could have moved down here and have a single block which would be for both `norvegicus` and `rat` as the "last resort".